src/map/battle.cpp

```diff
@@ -7192,9 +7192,9 @@ static void battle_calc_attack_plant(struct Damage* wd, struct block_list *src,s
 		return;
 	}
 
-	// Triple Attack and Finger Offensive have a special property, they do not split damage on plant mode
+	// Skills have a special property, they do not split damage on plant mode
```

Comment:
```suggestion
	// Some skills have a special property, they do not split damage on plant mode
```
Probably should add "Some" here so it doesn't sound like it goes for skills in general.